pkg/api/v1/events/handlers.go

```diff
@@ -122,7 +122,8 @@ func (h *Handler) ControllerEvent(ctx context.Context, msg events.Message) {
 	}
 }
 
-func (h *Handler) updateCondition(ctx context.Context, updEvt *v1types.ConditionUpdateEvent) error {
+// UpdateCondition sanity checks the incoming condition update, merges it and applies the result to serverservice
+func (h *Handler) UpdateCondition(ctx context.Context, updEvt *v1types.ConditionUpdateEvent) error {
```

Comment:
We may want to expose the bool parameter to have the `ResourceVersion` validated for updates from nats KV.

Reply:
This might be moot in light of my reply above. Let's come to a consensus.

Reply:
ack, I agree its not required.